packages/runtime-common/realm.ts

```diff
@@ -556,7 +562,10 @@ export class Realm {
 
   private async removeCardSource(request: Request): Promise<Response> {
     let localName = this.paths.local(request.url);
-    let handle = await this.getFileWithFallbacks(localName);
+    let handle = await this.getFileWithFallbacks(
+      localName,
+      executableExtensions,
```

Comment:
probably we should also handle DELETE requests in the same way: by including `.json` fallback